docs/source/private_data_tutorial.rst

```diff
@@ -285,7 +285,7 @@ For example, in the following snippet of the ``CreateAsset`` function,
         }
 
         type assetTransientInput struct {
-            Type           string `json:"objectType"` //Type is used to distinguish the various types of objects in 
+            Type           string `json:"objectType"` //Type is used to distinguish the various types of objects in
```

Comment:
Since you're fixing things
```
Type is used to distinguish the various types of objects in
```
I can't parse this sentence... could you pls fix it?

Reply:
Done `//Type is used to distinguish the various types of objects in state database`